src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -1520,20 +1520,21 @@ public override int GetHashCode()
         {
             if (IsNotAbsoluteUri)
             {
-                return CalculateCaseInsensitiveHashCode(OriginalString);
+                return OriginalString.GetHashCode();
```

Comment:
It no longer needs to be case-insensitive?

Reply:
I'm not sure how it showed up in the diff here as the change was made in #32713. The idea is that equality is only case-**in**sensitive for Unc and Dos paths, so there's no need for the more expensive hash-code calculation in all other cases.